src/listeners/ready.ts

```diff
@@ -1,11 +1,69 @@
-import { client } from "../index.js"
+import { client, Command } from "../index.js"
 import stats from "../events/stats.js"
 import inactives from "../events/inactives.js"
 import { listeningStatuses, watchingStatuses, playingStatuses } from "../config.json"
+import { ApplicationCommand, ApplicationCommandPermissionData, Collection } from "discord.js"
 
 client.once("ready", async () => {
     console.log("Ready!")
 
+    const publishCommand = async (command: Command) => {
+        if (command.allowDM) {
+            //Create a global command
+            var cmd = await client.application?.commands.create({
```

Comment:
Don’t use `var`
Also we can’t create the command every time, only the first time this update is published